src/schema.ts

```diff
@@ -314,6 +314,46 @@ export const applicationRelations = relations(applications, ({ many }) => ({
   accessTokens: many(accessTokens),
 }));
 
+export const accessGrants = pgTable(
+  "access_grants",
+  {
+    id: uuid("id").$type<Uuid>().primaryKey(),
+    token: text("token").notNull().unique(),
+    expiresIn: integer("expires_in").notNull(),
+    redirectUri: text("redirect_uri").notNull(),
+    scopes: scopeEnum("scopes").array().notNull(),
+    applicationId: uuid("application_id")
+      .$type<Uuid>()
+      .notNull()
+      .references(() => applications.id, { onDelete: "cascade" }),
+    resourceOwnerId: uuid("resource_owner_id")
+      .$type<Uuid>()
+      .notNull()
+      .references(() => accountOwners.id, { onDelete: "cascade" }),
+    codeChallenge: text("code_challenge"),
+    codeChallengeMethod: text("code_challenge_method"),
```

Comment:
We could arguably use an enum here since the value can only ever be `S256`, since we don't support `plain` and no other challenge methods are supporter.

Reply:
I'd also be okay with removing these until we add in support for PKCE.

Reply:
I think we might as well remove these as of now?

Reply:
Removed